spec/definitions.yaml

```diff
@@ -715,6 +715,13 @@ definitions:
             version:
               type: string
               description: Version number of the component
+      state:
+        type: string
```

Comment:
```suggestion
        type: string
        enum: ["active", "wip", "deprecated"]
```

Reply:
Please also add the `state` attribute to any response examples where it applies.